features/cli/display_failing_result.feature

```diff
@@ -6,5 +6,18 @@ Feature: Display failing result
     Then it should fail with:
       """
       ✗ http://localhost:54321/missing_header.html
-        - Found 0 h1 elements.
+        * Headings > Exactly one main heading
+          - Found 0 h1 elements.
+      """
+
+  Scenario: Two standards fail
+    Given a website running at http://localhost:54321
+    When I run `a11y http://localhost:54321/two_headings_failures.html`
+    Then it should fail with:
+      """
+      ✗ http://localhost:54321/two_headings_failures.html
+        * Headings > Content must follow headings
```

Comment:
Sure, colons.

Most other a11y tools don't have any whitespace in the name of the standard/sniff, it's more of an identifier, presumably so they can be ignored without any confusion over whitespace. Personally I quite like the humanised names, instead of something like `Headings.ContentMustFollowHeadinhgs`. WDYT?